index.html

```diff
@@ -234,5 +235,47 @@ <h1>
         </li>
       </ul>
     </section>
+    <section class="contact-me">
+      <div class="container">
+        <h3 class="contact-title">Contact me</h3>
+        <p class="contact-p">
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
+        <form
+          class="contact-form"
+          action="https://formspree.io/f/mvolqjoe"
+          method="POST"
+        >
+          <input
+            type="text"
+            name="user_name"
+            id="name"
+            placeholder="Yeremias"
+            maxlength="30"
+            required
+          />
+          <input
+            type="email"
+            name="user_email"
+            id="mail"
+            placeholder="you@you.com"
+            required
+          />
+          <textarea
+            name="user_message"
+            id="message"
+            cols="30"
+            rows="10"
+            maxlength="500"
+            placeholder="Write your message here"
+          ></textarea>
```

Comment:
- [ OPTIONAL] Please adjust your text color below, try to make it a little bit black than this actual color  
<img width="301" alt="visi" src="https://user-images.githubusercontent.com/79058364/159565756-8d420215-1f20-48ef-a4ee-9ac885b42bf0.png">